kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -121,10 +137,9 @@
   @require '~kolibri.styles.definitions'
 
   $card-width = 210px
-  $card-height = $card-width
   $card-thumbnail-ratio = (9 / 16)
-  $card-thumbnail-height = $card-width * $card-thumbnail-ratio
-  $card-text-height = $card-height - $card-thumbnail-height
+  $card-thumbnail-height = 100% * $card-thumbnail-ratio
```

Comment:
Making the thumbnail percentage based so that we can assign styles to the parent component